packaging/test_plugin_indexing.py

```diff
@@ -36,6 +36,10 @@
     long_type = long # Python 2.x
 except NameError: pass
 
+Manual_Tests_Enabled = (
+    os.environ.get("MANUALLY_TEST_INDEXING_PLUGIN","")[:1].lower() not in ("n", "f", "0", "")
+    )
```

Comment:
Is this documented anywhere?

Reply:
Yes, just did that. README section on Testing the Plugin